network/components/virt_rx.c

```diff
@@ -39,6 +41,18 @@ state_t state;
 /* Boolean to indicate whether a packet has been enqueued into the driver's free queue during notification handling */
 static bool notify_drv;
 
+bool is_broadcast(struct ethernet_header *buffer)
```

Comment:
I think it is cleaner to combine the MAC address checking functions into one function...

Reply:
There are also other broadcast MAC addresses other than 0xFFFFFFFFFFFF

Reply:
Hmm, I only thought that there were multiple broadcast addresses as when I was testing the sDDF I looked up some of the MAC addresses that our incoming packets were addressed to, and thought I remembered that they were differenc broadcast MAC addresses. But after looking them up again now, I seem to have gotten confused and these were actually _multicast_ addresses, so this isn't an issue for us. But in any case, now that we have reference counts we could easily support multicast if we ever needed to.